src/useCases/getAllUsers.ts

```diff
@@ -6,11 +6,15 @@ import UserFullDetails from "types/UserFullDetails"
 
 const getAllUsers = async (
   connection: Database,
-  page: number
+  visibleForces: string,
+  page = 0
 ): PromiseResult<PaginatedResult<Pick<UserFullDetails, "username" | "forenames" | "surname" | "emailAddress">[]>> => {
-  let users
-
-  const getAllUsersQuery = `
+  let users = []
+  if (visibleForces !== "") {
+    const forces = visibleForces.split(",")
+    /* eslint-disable @typescript-eslint/naming-convention */
+    const forceWhere = forces.map((code) => `visible_forces like '%${code}%'`).join(" OR ")
```

Comment:
I don't think `%${code}%` is tight enough. Imagine the case where we had force codes `10` and `100` - `%10%` would match both.

I think we could use a regex with a word boundary for this - i.e. `visible_forces ~ '\y10\y'` would only match `10` and not `100`

Reply:
ah yes, but force codes are all 3 characters long. Hence why I went with this approach

Reply:
I can look at making it more strict just in case